client-v2/src/shared/util/url.ts

```diff
@@ -13,4 +18,56 @@ function getAbsolutePath(url: string, keepQueryString?: boolean) {
   return keepQueryString ? path + a.search : path;
 }
 
-export { getAbsolutePath };
+/**
+ * Get the complete viewable URL for an article.
+ */
+export function getViewUrl(article: DerivedArticle) {
```

Comment:
Could we get rid of the elses in the function by returning the `url` ince we know it's value?

Reply:
We can get rid of this function completely, because it's not used! 🙃